plugins/transforms/ldap/src/main/java/org/apache/hop/pipeline/transforms/ldapoutput/LdapOutputDialog.java

```diff
@@ -1441,7 +1461,7 @@ private void generateMappings() {
       }
       message += Const.CR;
       message += BaseMessages.getString( PKG, "LdapOutputDialog.DoMapping.SomeFieldsNotFoundContinue" ) + Const.CR;
-      MessageDialog.setDefaultImage( GuiResource.getInstance().getImageHopUi() );
+      Window.setDefaultImage( GuiResource.getInstance().getImageHopUi() );
```

Comment:
MessageDialog --> IconAndMessageDialog --> Dialog --> Window

setDefaultImage is a static method from Window

Reply:
alright! thanks! Merging